staging/src/k8s.io/client-go/tools/portforward/portforward.go

```diff
@@ -379,9 +379,17 @@ func (pf *PortForwarder) handleConnection(conn net.Conn, port ForwardedPort) {
 	localError := make(chan struct{})
 	remoteDone := make(chan struct{})
 
+	drainRemoteStream := func() {
+		_, err := io.ReadAll(dataStream)
+		if err != nil {
+			fmt.Fprintf(pf.out, "error draining all remain data from remote stream after connection broken\n")
```

Comment:
should this call `runtime.HandleError` like other error paths in this function instead of `fmt.Fprintf(pf.out`?

Reply:
also, if we only get here when an error has already occurred, do we really need to double error?

Reply:
replaced with `runtime.HandleError`.